pkg/utils/common.go

```diff
@@ -173,6 +173,12 @@ func ConvertCommitHashToShort(hash string) string {
 	return hash[:7]
 }
 
+func BuildCommitURL(url string, commitHash string) string {
```

Comment:
Does it make sense to use a `NormalizedRepoURL` here?

Reply:
I thought about that and I think when we have gitlab it will make sense. I was kinda hoping my talks with the go-git-provider team to fix the logr issues would be done before we need to deal with gitlab.